pkg/typesystem/typesystem.go

```diff
@@ -12,7 +12,6 @@ import (
 
 	"github.com/emirpasic/gods/sets/hashset"
 	"go.opentelemetry.io/otel"
-	"gonum.org/v1/gonum/graph/topo"
```

Comment:
FYI this is my end goal. `openfga` server shouldn't know about how the auth model graph is implemented under the hood.